File: app/src/main/java/com/service/saver/saverservice/folder/model/FileModel.java
package com.service.saver.saverservice.folder.model;

import java.io.Serializable;

public class FileModel implements Serializable {
    private Long id;
    private String name;
    private String filepath;

    public FileModel() {
    }

    public FileModel(Long id, String name, String filepath) {
        this.id = id;
        this.name = name;
        this.filepath = filepath;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getFilepath() {
        return filepath;
    }

    public void setFilepath(String filepath) {
        this.filepath = filepath;
    }


}

File: app/src/main/java/com/service/saver/saverservice/folder/model/FileModel.kt
package com.service.saver.saverservice.folder.model

import java.io.Serializable

class FileModel : Serializable {
    var id: Long? = null
    var name: String? = null
    var filepath: String? = null
    var isFolder: Boolean = false
    var parent: String = ""
    constructor(id: Long?, name: String, filepath: String, b: Boolean, parent: String) {
        this.id = id
        this.name = name
        this.filepath = filepath
        this.isFolder = b
        this.parent= parent
    }
}
